mailpoet/assets/js/src/automation/integrations/mailpoet/steps/clicks-email-link/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { __ } from '@wordpress/i18n';
+import { StepType } from '../../../../editor/store';
+import { LockedBadge } from '../../../../../common/premium-modal/locked-badge';
+import { PremiumModalForStepEdit } from '../../../../../common/premium-modal';
+import { Icon } from './icon';
+
+const keywords = [
+  __('email', 'mailpoet'),
+  __('link', 'mailpoet'),
+  __('click', 'mailpoet'),
```

Comment:
Very minor, but this might be better as `clicks`. Kind of weird for the trigger to disappear from search results going from `click` to `clicks`.

Reply:
Good catch: https://github.com/mailpoet/mailpoet/pull/5255/commits/50aec33da7dc4e46fa47737b573978a986b94362